### PR TITLE
Add `T_min_ref` and bump version

### PR DESCRIPTION
**Pull request by charleskawczynski**
- Bumps version
 - Exports some parameters
 - Adds `T_min_ref`

### CODE REVIEW

File: src/Planet.jl [hunk 1]
```diff
@@ -42,7 +42,10 @@ export molmass_dryair,
     year_anom,
     orbit_semimaj,
     TSI,
-    MSLP
+    MSLP,
+    T_surf_ref,
+    ΔTv_ref,
```
Comment (tapios) [inline src/Planet.jl:47]:
```suggestion
```
(As I suggested in the other PR, I'd remove `ΔTv_ref` here and instead replace this by `ΔTv_ref = T_surf_ref - T_min_ref` in the relevant code.)

File: src/PlanetParameters.jl [hunk 2]
```diff
@@ -47,4 +47,5 @@ Planet.TSI(ps::AbstractEarthParameterSet)            = 1362
 Planet.MSLP(ps::AbstractEarthParameterSet)           = 1.01325e5
 Planet.T_surf_ref(ps::AbstractEarthParameterSet)     = 290.0
 Planet.ΔTv_ref(ps::AbstractEarthParameterSet)        = 60.0
```
Comment (tapios) [inline src/PlanetParameters.jl:49]:
```suggestion
```

File: src/Planet.jl (hunk 1 above)
Comment (charleskawczynski) [reply src/Planet.jl:47]: > in the relevant code.

In CLIMAParameters.jl? Or in ClimateMachine.jl

File: src/PlanetParameters.jl (hunk 2 above)
Comment (charleskawczynski) [reply src/PlanetParameters.jl]: I've modified this for now to
```julia
Planet.ΔTv_ref(ps::AbstractEarthParameterSet)        = Planet.T_surf_ref(ps) - Planet.T_min_ref(ps)
```
Waiting to hear back regarding [this question](https://github.com/CliMA/CLIMAParameters.jl/pull/20#discussion_r423896240) on how to proceed.

**Pull request merged**